web/src/rendered_markdown.ts

```diff
@@ -353,4 +353,10 @@ export const update_elements = ($content: JQuery): void => {
                 $inline_img_thumbnail.attr("src")!.replace(/\/[^/]+$/, "/" + thumbnail_name),
             );
         });
+
+    $content
+        .find("div.message_inline_image img, div.message_inline_image video")
+        .each(function (): void {
+            parse_media_data(this);
+        });
```

Comment:
What's the thinking around calling this here? I assume it's cheap, but I think a comment explaining the purpose of doing it might be helpful -- is part of the motivation to update our media list cache?

Reply:
I double-checked and this looks to be unnecessary; I think it was necessary in an earlier version.  Removing it.